asset/css/markbind.css

```diff
@@ -116,11 +116,8 @@ footer {
 /* Side Navigation menus */
 
 .spacer-top {
-    /*
-      spacer-top class is a work around for MarkBind <navbar> component using "position: fixed".
-      This class can be removed once the component's position type is changed.
-    */
-    padding-top: 70px;
+    /* This class is to ensure adequate spacing of contents from the top of the page */
```

Comment:
Just wondering - should this comment be placed outside the class definition (even though the comment was previously inside the class definition)? 🙂

Reply:
For this section, how I formatted it was 

```CSS
/**
  * Main section description
  */

/* sub section one */

/* sub section two */

.some-class {
  /* class description */
}
```

My intention was to separate sub section title comments from class descriptions